ProfileService/ProfileService/DI/ServicesConfiguration.cs

```diff
@@ -0,0 +1,79 @@
+﻿using AutoMapper;
+using Microsoft.AspNetCore.Authentication.JwtBearer;
+using Microsoft.IdentityModel.Tokens;
+using ProfileService.BLL.Utilities.Mapping;
+using ProfileService.DAL.Repositories.IRepositories;
+using ProfileService.DAL.Repositories;
+using System.Security.Claims;
+using ProfileService.BLL.Services.IServices;
+using ProfileService.BLL.Services;
+using Microsoft.EntityFrameworkCore;
+using ProfileService.DAL.DataContext;
+
+namespace ProfileService.DI
+{
+    public static class ServicesConfiguration
+    {
+        public static void AddDatabaseContext(this IServiceCollection services, IConfiguration configuration) => services.AddDbContext<ProfileDbContext>(options =>
+        {
+            var connectionString = configuration["ConnectionStrings:ProfileDatabase"];
```

Comment:
You can use configuration.GetConnectionString("ProfileDatabase") extension method